HelpSource/Guides/SynthDefsVsSynths.schelp

```diff
@@ -39,7 +39,7 @@ SynthDef("Help-SynthDef", { arg out = 0;
 (
 SynthDef("Help-SynthDef", { arg out = 0;
 	Out.ar(out, PinkNoise.ar(0.1))
-}).send(s);
+}).add;
```

Comment:
here also not, because it is part of the explanation. You could add one example and explanation with `add` that would be good.

Reply:
Thank you, and my apologies for the change. I have reverted it and included an `.add` example in https://github.com/supercollider/supercollider/pull/7117/commits/c20a7517dfb86150b5cece43e99ac03f37bd0e1b.

I have also restructured the text and added further explanation. I am not certain whether this will be acceptable; perhaps @telephon, @HotwheelsSisyphus, and @tedmoore might wish to review this change in detail.

Reply:
I regard this as resolved since this is outdated!